lang/cs/Org.Apache.REEF.Client/AzureBatch/Service/AzureBatchService.cs

```diff
@@ -39,21 +40,39 @@ public sealed class AzureBatchService : IDisposable
         public string PoolId { get; private set; }
 
         private BatchClient Client { get; set; }
+        private ContainerRegistryProvider ContainerRegistryProvider { get; set; }
+        private IList<string> Ports { get; set; }
+        private ICommandBuilder CommandBuilder { get; set; }
+        private bool AreContainersEnabled
```

Comment:
just
```C#
private bool AreContainersEnabled => ContainerRegistryProvider.IsValid();
```

Reply:
Fixed.

---
In reply to: [203900019](https://github.com/apache/reef/pull/1478#discussion_r203900019) [](ancestors = 203900019)